src/sap/fhir/model/r4/FHIRFilterOperatorUtils.js

```diff
@@ -137,5 +141,101 @@ sap.ui.define(["sap/fhir/model/r4/FHIRFilterOperator", "sap/fhir/model/r4/FHIRFi
 		return sFHIRSearchPrefix;
 	};
 
+	/**
+	 * Transforms the UI5 filter operator to a FHIR valid filter prefix based on the given UI5 <code>oFilter</code>
+	 *
+	 * @param {sap.ui.model.Filter} oFilter The given filter
+	 * @returns {string} The FHIR filter prefix
+	 * @protected
+	 * @since 2.1.0
+	 */
+	FHIRFilterOperatorUtils.getFHIRFilterPrefix = function (oFilter) {
+		var sFHIRFilterPrefix;
+		switch (oFilter.sOperator) {
+			case FHIRFilterComplexOperator.EQ:
+				sFHIRFilterPrefix = "eq";
+				break;
+			case FHIRFilterComplexOperator.NE:
+				sFHIRFilterPrefix = "ne";
+				break;
+			case FHIRFilterComplexOperator.GT:
+				sFHIRFilterPrefix = "gt";
+				break;
+			case FHIRFilterComplexOperator.GE:
+				sFHIRFilterPrefix = "ge";
+				break;
+			case FHIRFilterComplexOperator.LT:
+				sFHIRFilterPrefix = "lt";
+				break;
+			case FHIRFilterComplexOperator.LE:
+				sFHIRFilterPrefix = "le";
+				break;
+			case FHIRFilterComplexOperator.SA:
+				sFHIRFilterPrefix = "sa";
+				break;
+			case FHIRFilterComplexOperator.EB:
+				sFHIRFilterPrefix = "eb";
+				break;
+			case FHIRFilterComplexOperator.AP:
+				sFHIRFilterPrefix = "ap";
+				break;
+			case FHIRFilterComplexOperator.StartsWith:
+				sFHIRFilterPrefix = "sw";
+				break;
+			case FHIRFilterComplexOperator.EndsWith:
+				sFHIRFilterPrefix = "ew";
+				break;
+			case FHIRFilterComplexOperator.Contains:
+				sFHIRFilterPrefix = "co";
+				break;
+			case FHIRFilterComplexOperator.PR:
+				sFHIRFilterPrefix = "pr";
+				break;
+			case FHIRFilterComplexOperator.PO:
+				sFHIRFilterPrefix = "po";
+				break;
+			case FHIRFilterComplexOperator.SS:
+				sFHIRFilterPrefix = "ss";
+				break;
+			case FHIRFilterComplexOperator.SB:
+				sFHIRFilterPrefix = "sb";
+				break;
+			case FHIRFilterComplexOperator.IN:
+				sFHIRFilterPrefix = "in";
+				break;
+			case FHIRFilterComplexOperator.NI:
+				sFHIRFilterPrefix = "ni";
+				break;
+			case FHIRFilterComplexOperator.RE:
+				sFHIRFilterPrefix = "re";
+				break;
+			default:
+				break;
+		}
+		return sFHIRFilterPrefix;
+	};
+
+	/**
+	 * Parses the JS filter value to a FHIR filter value
+	 *
+	 * @param {string} sFilterValue The value type of a filter object
+	 * @param {any} vValue The value of a filter object
+	 * @returns {string} Formatted FHIR filter value
+	 * @protected
```

Comment:
You're right. when getFilterValue is public than this method should also be public